charts/trento-server/Chart.yaml

```diff
@@ -1,13 +1,13 @@
 # SPDX-License-Identifier: Apache-2.0
-#!BuildTag: trento/trento-server:1.1.0
-#!BuildTag: trento/trento-server:1.1.0-build%RELEASE%
+#!BuildTag: trento/trento-server:1.1.1
+#!BuildTag: trento/trento-server:1.1.1-build%RELEASE%
 apiVersion: v2
 name: trento-server
 description: The trento server chart contains all the components necessary to run a Trento server.
 type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates
-version: 1.1.0
+version: 1.1.1
```

Comment:
I think we discussed in the past that we would use development suffixes and only bump to release versions when doing actual releases.

Reply:
see https://semver.org/#spec-item-9

Reply:
true, fixed